Documentation/Branching.md

```diff
@@ -90,11 +126,13 @@ Holds minimal changes for the purpose of making the upcoming release more stable
    <div>
       <span style="font-weight:bold">Branching Diagram</span>
    </div>
-   <a href="#"><img style="width: 75%" align="center" src="./Images/git-branching-diagram.jpg"></a>
+
+![BranchingDiagram](./Images/BranchingDiagram-DarkMode-v1.1.png#gh-dark-mode-only)
+![BranchingDiagram](./Images/BranchingDiagram-LightMode-v1.1.png#gh-light-mode-only)
```

Comment:
@kselena 
I was reviewing the document by rendering it as markdown and I noticed that these images do not exist in the project.
Because they do not exist, they end up as a broken image link in the markdown.  They simply need to be added.

The files can be found in the **Velaptor** project.